coast/__init__.py

```diff
@@ -8,6 +8,5 @@
 from .CDF import CDF
 from .CRPS import CRPS
 from .interpolate_along_dimension import interpolate_along_dimension
-from .STATS import STATS
+from .package import PACKAGE
```

Comment:
Line 11: I don't think that you want PACKAGE? I commented it out to get it to work.